### DISCUSSION TITLE
1750E Update sources patch

### DISCUSSION THREAD

**Discussion by SaMMy-lacht** [Freetz-NG/freetz-ng#807] [Allgemein]
Keine Lust auf ein Pull zu erstellen ;-)

[1750_07.30.zip](https://github.com/Freetz-NG/freetz-ng/files/11755665/1750_07.30.zip)

**Comment by fda77** [comment]
Danke, aber dies ist recht überflüssig... Wenn man die integrierte 7.27 und die neue 7.30 vergleicht ist dies die KOMPLETTE Änderung am Kernel:
```
diff -dauNr --no-dereference --color 727/GPL/x/linux/.kernelvariables 730/GPL/x/linux/.kernelvariables
--- 727/GPL/x/linux/.kernelvariables    2021-07-06 16:17:12.000000000 +0200
+++ 730/GPL/x/linux/.kernelvariables    2023-06-13 15:09:20.000000000 +0200
@@ -1,6 +1,6 @@
 # vim: set autoread filetype=make:
 #
-# Auto-generated at build-89370-dirty
+# Auto-generated at build-100771-dirty
 # while generating for Fritz_Box_HW206
 #
 ARCH = mips
```

Man muss übrigens nicht immer ein komplett neues Archiv erstellen, man kann auch einen delta-patch wie `make/kernel/patches/*/*/*delta*` packen, aber selbst dafür ist die Änderung zu irrelevant :)
